### PR TITLE
Skills qtk, wt, xw-widgets added and png to svg skills

### PR DESCRIPTION
**Pull request by MauricioHernanCabrera**
https://github.com/rahuldkjain/github-profile-readme-generator/issues/78

### CODE REVIEW

File: src/constants/skills.js [hunk 1]
```diff
@@ -343,6 +346,9 @@ const icons = {
   puppeteer: "https://i.ibb.co/Qk299CX/puppeteer.png",
   karma: "https://i.ibb.co/dbgh2DH/jasmine.png",
   jasmine: "https://i.ibb.co/55txF2S/karma.png",
+  gtk: "https://i.ibb.co/JyGnDrY/gtk.png",
```
Comment (rahuldkjain) [inline src/constants/skills.js:349]: @MauricioHernanCabrera Please use SVG icons only. I will remove the present png icons too

**Review by rahuldkjain — CHANGES_REQUESTED** [review]

File: src/constants/skills.js (hunk 1 above)
Comment (MauricioHernanCabrera) [reply src/constants/skills.js:349]: Okasss

Comment (MauricioHernanCabrera) [reply src/constants/skills.js:349]: @rahuldkjain if you want, I can change the others skills of png to svg

**Review by rahuldkjain — APPROVED** [review]
Amazing @MauricioHernanCabrera . Thanks for the amazing contributions to the tool🙇.

**Pull request merged**